### PR TITLE
GA4: Set up basic lambda context [INTEG-92]

### PR DESCRIPTION
**Pull request by jsdalton**
Installs a basic lambda context and scopes the current app to a dedicated `frontend/` directory.

## Purpose

We will need a lambda backend to proxy API request to the Google Analytics API securely. This is because the credentials are secret and are shared for a given app installation, and thus we cannot make requests directly from the frontend.

To make review of PRs easier this big setup chunk of work is done in this PR.

## Approach

* Just tried to keep this as vanilla as possible. It's just an express app with standard tooling etc.
* No serverless stuff is installed yet
* There's a health endpoint more just as a sanity check. As a concept this seems like it's not useful in a Lambda

### CODE REVIEW

File: apps/google-analytics-4/lambda/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+const port = 3000;
+
+app.listen(port, () => {
+  return console.log(`Express server is listening at http://localhost:${port}`);
```
Comment (shikaan) [inline apps/google-analytics-4/lambda/src/index.ts:6]: nit: no need to return void here

**Review by shikaan — COMMENTED** [review]
Looks good, you may want to add a `serverless.yaml` file (or another manifest, if you want to go in another direction) to host the lambda. Serverless in particular comes with a few utilities that help you develop locally as well, without having to tinker with the setup too much

File: apps/google-analytics-4/lambda/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lambda",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```
Comment (kdamball) [inline apps/google-analytics-4/lambda/package.json:6]: Missing deploy commands for this lambda

**Review by kdamball — COMMENTED** [review]
Some deployment commands are missing. Additionally, the frontend is missing the app def id in `upload` command. It's fine to hardcode the app def id there as it's public anyway. We do something similar in other apps

**Comment by jsdalton** [comment]
THanks @kdamball and @shikaan for the reviews. I have a separate ticket for doing the lambda and deployment work you both called out. @kdamball I also created a separate task for the frontend deployment. Basically, I just hadn't gotten that far yet.

**Review by kdamball — APPROVED** [review]

**Pull request merged**